papis/arxiv.py

```diff
@@ -83,7 +83,7 @@ def get_data(
     xmldoc = urllib.request.urlopen(url).read()
 
     import bs4
-    soup = bs4.BeautifulSoup(xmldoc, 'html.parser')
+    soup = bs4.BeautifulSoup(xmldoc, features='lxml-xml')
```

Comment:
There was a warning from `bs4` related to using a HTML parser for an XML document here, and this seemed to force the `lxml` version. At least on my machine using just `lxml` found the `lxml-html` parser and still complained.. not quite sure what that's about..